src/main/java/com/linecorp/armeria/client/circuitbreaker/EventCounter.java

```diff
@@ -25,12 +27,18 @@
 
     /**
      * Counts success event
+     *
+     * @return An {@link Optional} containing the current {@link EventCount} if it has been updated, or else
+     * an empty {@link Optional}.
      */
-    void onSuccess();
+    Optional<EventCount> onSuccess();
```

Comment:
`SlidingWindowCounter` does not assume too short interval. It would also cause other problems (too many internal buckets will be created, etc)